alerting/src/main/kotlin/org/opensearch/alerting/script/BucketLevelTriggerExecutionContext.kt

```diff
@@ -5,21 +5,25 @@
 
 package org.opensearch.alerting.script
 
+import org.apache.logging.log4j.LogManager
+import org.opensearch.alerting.model.AlertContext
 import org.opensearch.alerting.model.BucketLevelTriggerRunResult
 import org.opensearch.alerting.model.MonitorRunResult
 import org.opensearch.commons.alerting.model.Alert
 import org.opensearch.commons.alerting.model.BucketLevelTrigger
 import org.opensearch.commons.alerting.model.Monitor
 import java.time.Instant
 
+private val logger = LogManager.getLogger(BucketLevelTriggerExecutionContext::class.java)
+
 data class BucketLevelTriggerExecutionContext(
     override val monitor: Monitor,
     val trigger: BucketLevelTrigger,
     override val results: List<Map<String, Any>>,
     override val periodStart: Instant,
     override val periodEnd: Instant,
     val dedupedAlerts: List<Alert> = listOf(),
-    val newAlerts: List<Alert> = listOf(),
+    val newAlerts: List<AlertContext> = listOf(),
```

Comment:
could have been a separate field to have alert context list and retain new alerts

Reply:
I'll add this as a follow-up item.